### PR TITLE
docs: add changelog for mar prerelease

### PR DESCRIPTION
**Pull request by MuyangAmigo**

### CODE REVIEW

File: packages/vscode-extension/PRERELEASE.md [hunk 1]
```diff
@@ -2,6 +2,40 @@
...
+
+#### Enhancements
+
+- Updated the default app icon in the Teams Toolkit-generated app templates and samples with Microsoft 365 and Copilot-themed colors.
+- Added `LLM.Description` in the app manifest for bot-based message extensions when used as copilot plugin for better reasoning with LLMs. To utilize this feature, please enable the `Develop Copilot Plugin` feature setting via Visual Studio Code in the [User and Workspace Settings](https://code.visualstudio.com/docs/getstarted/settings) and create a new app via `Create a New App` -> `Message Extension` -> `Custom Search Results` -> `Start with Bot`.
+- Improved Azure account authentication with a built-in Microsoft authentication provider in Visual Studio Code. This enhancement increases the reliability of Azure authentication, especially when using a proxy.
+- Upgraded `Custom Search Results` (Start with a New API) template to Azure Functions to v4, the officially recommended version with better support. See more details for [Azure Functions runtime versions overview](https://learn.microsoft.com/azure/azure-functions/functions-versions?tabs=isolated-process%2Cv4&pivots=programming-language-javascript).
```
Comment (sffamily) [inline packages/vscode-extension/PRERELEASE.md:23]: "to v4" -> v4

File: packages/vscode-extension/PRERELEASE.md [hunk 2]
```diff
@@ -2,6 +2,40 @@
...
+- Multiple parameters are now supported for API-based message extensions.
+- Updated `Teams Chef Bot` sample to point to the latest version under [teams-ai repository](https://github.com/microsoft/teams-ai/tree/main/js/samples/04.ai.a.teamsChefBot).
```
Comment (sffamily) [inline packages/vscode-extension/PRERELEASE.md:25]: recommend to remove: "to point to the latest version"

File: packages/vscode-extension/PRERELEASE.md [hunk 3]
```diff
@@ -2,6 +2,40 @@
...
+- Fixed an issue where `arm/deploy.UnhandledError` might appear. [#10911](https://github.com/OfficeDev/TeamsFx/pull/10911)
+- Fixed an issue with inconsistent capitalizations in the project creation dialog. [#10792](https://github.com/OfficeDev/TeamsFx/pull/10792)
+- Fixed an issue with Teams Toolkit CLI where `Error: TeamsfxCLI.CannotDetectRunCommand` might appear when using the `teamsapp preview` command. [#10808](https://github.com/OfficeDev/TeamsFx/pull/10808)
+- Fixed an issue with unclear error messages when sideloading the app using an unsupported file format. [#10799](https://github.com/OfficeDev/TeamsFx/pull/10799)
+- Fixed an issue where an unexpected error might occur when executing `teamsapp account login azure`. [#11015](https://github.com/OfficeDev/TeamsFx/pull/11015)
+- Fixed broken links in README documentation. [#10836](https://github.com/OfficeDev/TeamsFx/pull/10836), [#10831](https://github.com/OfficeDev/TeamsFx/pull/10831)
+- Fixed an issue where featured sample are not shown in the full list. [#10841](https://github.com/OfficeDev/TeamsFx/pull/10841)
```
Comment (sffamily) [inline packages/vscode-extension/PRERELEASE.md:36]: sample -> samples

File: packages/vscode-extension/CHANGELOG.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 # Changelog
 
-> Note: This changelog only includes the changes for the stable versions of Teams Toolkit. For the changelog of Prereleased versions, please refer to the [Teams Toolkit Prerelease Changelog](./PRERELEASE.md).
+> Note: This changelog only includes the changes for the stable versions of Teams Toolkit. For the changelog of prereleased versions, please refer to the [Teams Toolkit Prerelease Changelog](https://github.com/OfficeDev/TeamsFx/blob/dev/packages/vscode-extension/PRERELEASE.md).
```
[the remark this answers is not in the document]
Comment (MuyangAmigo) [reply packages/vscode-extension/CHANGELOG.md]: Updated

File: packages/vscode-extension/PRERELEASE.md [hunk 5]
```diff
@@ -2,6 +2,40 @@
...
 
+> Note: This changelog only includes the changes for the prerelease versions of Teams Toolkit. For the changelog of stable versions, please refer to the [Teams Toolkit Changelog](https://github.com/OfficeDev/TeamsFx/blob/dev/packages/vscode-extension/CHANGELOG.md).
+
+### March 19, 2024
+
+#### New Features
+
+- **Build Your Own Copilots in Teams with Teams AI Library**
+![Custom Copilots](https://github.com/OfficeDev/TeamsFx/assets/11220663/0387a2ce-ec39-4c72-aabc-1ec2b9e85d59)
+We have enhanced the user experience for developers to create their own copilots, an AI-powered intelligent chatbot for Teams, with the following improvements:
```
[the remark this answers is not in the document]
Comment (MuyangAmigo) [reply packages/vscode-extension/PRERELEASE.md]: Updated

File: packages/vscode-extension/PRERELEASE.md (hunk 1 above)
Comment (MuyangAmigo) [reply packages/vscode-extension/PRERELEASE.md]: Updated

File: packages/vscode-extension/PRERELEASE.md [hunk 6]
```diff
@@ -2,6 +2,40 @@
 
 ## Changelog
 
+> Note: This changelog only includes the changes for the prerelease versions of Teams Toolkit. For the changelog of stable versions, please refer to the [Teams Toolkit Changelog](https://github.com/OfficeDev/TeamsFx/blob/dev/packages/vscode-extension/CHANGELOG.md).
```
[the remark this answers is not in the document]
Comment (MuyangAmigo) [reply packages/vscode-extension/PRERELEASE.md]: Updated

File: packages/vscode-extension/PRERELEASE.md (hunk 2 above)
Comment (MuyangAmigo) [reply packages/vscode-extension/PRERELEASE.md]: Updated

File: packages/vscode-extension/PRERELEASE.md (hunk 3 above)
Comment (MuyangAmigo) [reply packages/vscode-extension/PRERELEASE.md]: Updated

**Review by sffamily — APPROVED** [review]

**Pull request merged**